### PR TITLE
Archivesspace delete

### PR DESCRIPTION
**Pull request by Hwesta**
Add delete functionality, with confirm before delete. Add a reload button to help keep SIP Arrange & ArchivesSpace in sync while testing.

Started adding notes & dates to the edit ArchivesSpace controller, but they mysteriously didn't show up.

Includes #93

### CODE REVIEW

File: app/arrangement/arrangement.controller.js [hunk 1]
```diff
@@ -59,6 +70,7 @@
 
         var idx = parent.indexOf(element);
         parent.splice(idx, 1);
+        vm.selected = undefined;
```
Comment (mistydemeo) [inline app/arrangement/arrangement.controller.js:73]: Good catch!

**Comment by mistydemeo** [comment]
> Started adding notes & dates to the edit ArchivesSpace controller, but they mysteriously didn't show up.

Is it possible your browser is caching the `archivesspace/form.html` partial? I had the issue with your changes not showing up until I cleared my cache.

**Comment by Hwesta** [comment]
> Is it possible your browser is caching the archivesspace/form.html partial? I had the issue with your changes not showing up until I cleared my cache.

Ah, maybe. I did a hard refresh a few times and wasn't seeing a GET for archivesspace/form.html but didn't otherwise clear my cache.

**Comment by mistydemeo** [comment]
Merged all but the dates/notes commit.

**Comment by mistydemeo** [comment]
Closing in favour of #95.

**Pull request closed without merging**